src/pages/bug/index.tsx

```diff
@@ -49,7 +49,7 @@ export default function BugPage() {
         onClick={() => {
           axios
             .post(
-              "https://world.openfoodfacts.org/api/v3/product/123456789",
+              `${OFF_API_URL_V3}product/123456789`,
```

Comment:
```suggestion
              `${OFF_API_URL_V3}/product/123456789`,
```

Reply:
All the other reference of `OFF_API_URL_V3` have this same pattern. But with the following def I assume the `/` is missing

```js
export const OFF_API_URL_V3 = `${OFF_URL}/api/v3`;
```